src/test/java/de/tum/in/www1/artemis/exam/StudentExamIntegrationTest.java

```diff
@@ -973,8 +975,6 @@ private void assertStudentExam1HasSingleTextSubmissionWithTextAndIsSubmitted(Str
     @Test
     @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
     void testSubmitStudentExam_notInTime() throws Exception {
-        studentExam1.setSubmitted(false);
-        studentExamRepository.save(studentExam1);
         // Forbidden because user tried to submit before start
         exam1.setStartDate(ZonedDateTime.now().plusHours(1));
         examRepository.save(exam1);
```

Comment:
Since the control flow of the respective method in the StudentExamRessource is very simple, we can be certain that the studentExam has not changed when we receive the HttpStatus forbidden and it is not necessary to include an additional check here.